<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Game</class>
 <widget class="QWidget" name="Game">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>598</width>
    <height>465</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(60, 141, 190);</string>
  </property>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>110</y>
     <width>161</width>
     <height>221</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
background-color:rgb(0,0,0);
</string>
   </property>
   <property name="maximum">
    <number>20</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="textVisible">
    <bool>false</bool>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar_2">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>110</y>
     <width>161</width>
     <height>221</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0,0,0);</string>
   </property>
   <property name="maximum">
    <number>20</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="textVisible">
    <bool>false</bool>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <property name="invertedAppearance">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>370</y>
     <width>158</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="button3">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(44,255,255)</string>
      </property>
      <property name="text">
       <string>I</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="button4">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(50,240,50)</string>
      </property>
      <property name="text">
       <string>P</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>370</y>
     <width>158</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="button1">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(250,200,50)</string>
      </property>
      <property name="text">
       <string>A</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="button2">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 85, 255);</string>
      </property>
      <property name="text">
       <string>D</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="start">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>350</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>30</y>
     <width>91</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <zorder>layoutWidget</zorder>
  <zorder>start</zorder>
  <zorder>label</zorder>
  <zorder>progressBar</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>progressBar_2</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
